content/articles/build-digital-clock-using-javascript/index.md

```diff
@@ -321,51 +273,48 @@ With what we have now, our code should work but we need to add style to it to ma
 We need to create a CSS file and add these styles. I call mine ```index.css```.
 
 ```css
-
 * {
```

Comment:
In this CSS code snippet, you used the body selector multiple times. Is this an error and you meant to style three different things? Please have a look at this and clarify it to me. If this was on purpose it might make more sense to have them under a single CSS rule.

Reply:
> In this CSS code snippet, you used the body selector multiple times. Is this an error and you meant to style three different things? Please have a look at this and clarify it to me. If this was on purpose it might make more sense to have them under a single CSS rule.

I have corrected that and put all the style inside one body but I haven't committed yet. Since I'm still working in the plagiarism check. Thank you @john-amiscaray